chapters/tutorials/tutorial_01.rst

```diff
@@ -0,0 +1,145 @@
+==================================
+Tutorial 1
+==================================
+
+.. contents:: Table of Contents
+    :depth: 2
+
+What is Jina
+----------------------------------
+
+Tutorial
+----------------------------------
+
+At the end of this tutorial, you will have your search engine using text. You will use text as an input and get a matching text for that.
+For this example, we will use the `Kaggle Wikipedia Corpus <https://www.kaggle.com/mikeortman/wikipedia-sentences>`_
+You will understand how every part of this example works and how you could create new apps with different datasets on your own.
+
+Set-up & overview
+----------------------------------
+
+The first part is to install Jina.
+
+  .. code-block:: python
+
+    pip install jina
+
+And we will also need the following dependencies:
+
+  .. code-block:: python
+
+    pip install click==7.1.2
+    pip install transformers==4.1.1
+    pip install torch==1.7.1
+
+
```

Comment:
For more information on installing Jina, refer to this page. https://docs.jina.ai/chapters/install/os/via-pip 

This is where we can cover things like edge cases, hardware issues, installing on windows.